### PR TITLE
fix: show tensorboard sources for CLI deployed tensorboards [DET-4372]

### PR DESCRIPTION
**Pull request by hkang1**
## Description

When starting a tensorboard with a single trial the tensorboard sources for experiments show up as an empty array instead of a `null` when it is created from the CLI (instead of through the API/WebUI). Handle the empty array cases too.

2020-10-14 at 11 03 AM

UPDATE: added the ability to show both experiment and trial type sources:

Screen Shot 2020-10-14 at 3 07 49 PM

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [x] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234

### CODE REVIEW

File: webui/react/src/pages/TaskList.tsx [hunk 1]
```diff
@@ -155,11 +155,11 @@ const TaskList: React.FC = () => {
         plural: '',
         sources: [] as number[],
       };
-      if (record.misc?.experimentIds) {
+      if (record.misc?.experimentIds && record.misc?.experimentIds.length !== 0) {
```
Comment (hamidzr) [inline webui/react/src/pages/TaskList.tsx:158]: suggestion: normalize in the decoder to make sure it's always a list and just check the length here.

**Review by hamidzr — APPROVED** [review]
LGTM. 

Not the goal of this pr: But the code around the sources is assuming that we would be only having trials or experiments as sources and not both. maybe we could show both? is that something we want to support?  cc @vishnu2kmohan

**Comment by hkang1** [comment]
@hamidzr there are enough changes in there I think is worth another look 🙏

**Comment by hamidzr** [comment]
does this need release notes?

**Review by hamidzr — APPROVED** [review]
LG.

Screen Capture_select-area_20201014150145

**Comment by apizzini** [comment]
I think this looks good, I'm just still a bit ignorant about some logics and namings in the app, so cannot be 100% sure about all the implications of some changes.

**Pull request merged**